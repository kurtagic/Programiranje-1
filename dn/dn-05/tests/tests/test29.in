41
7 97
14 84
7 92
8 95
3 87
1 95
11 92
7 82
8 91
0 90
14 90
3 93
1 79
1 98
2 90
0 84
3 100
14 83
14 92
14 88
2 98
7 99
1 99
14 80
8 93
11 97
1 86
0 95
7 99
7 97
11 86
0 79
3 80
14 85
2 92
11 87
3 87
1 81
11 94
1 98
8 82
