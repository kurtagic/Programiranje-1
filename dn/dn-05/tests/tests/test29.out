79: 6
80: 7
81: 12
82: 8
83: 8
84: 9
85: 12
86: 11
87: 19
88: 20
89: 19
90: 14
91: 13
92: 15
93: 15
94: 16
95: 18
96: 15
97: 18
98: 20
99: 17
100: 15
101: 2
102: 1
